### ISSUE TITLE
Indent makes `<style>` tag render incorrectly

### ISSUE THREAD

**Issue by bjmin** [probablyup/markdown-to-jsx#172]
I've tried to add `<style />` html tag to markdown

I supposed codes below
```
<style>
.bar {
  font-size: 12px;
}
</style>
```
should render properly

but turns out it will add `<p />` between lines, which results in
```
<style>
  <p>
    .bar {
      font-size: 12px;
    }
  </p>
</style>
```

but if I remove indent before `font-size: 12px`, it works again.

any ideas? using v6.6.1

**Comment by probablyup** [comment]
Good catch. Yeah that inside content should not be processed. Will add it
to my list

**Comment by probablyup** [comment]
Fixed in `6.6.2`, thanks for reporting!

**Comment by bjmin** [comment]
thx!
